.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -8,7 +8,7 @@ about: Create a report to help us improve.
  - **WebdriverIO version:** [e.g. 4.13.2]
  - **Mode:** [Standalone mode or WDIO Testrunner]
  - **If WDIO Testrunner, running sync/async:** [e.g. sync/async]
- - **Node.js version: [e.g. 8.9.1]
+ - **Node.js version:** [e.g. 8.9.1]
```

Comment:
Since you are at it - would you mind updating the example version to 8.11.2, since thats gonna be the min version for v5?